xml/security_ldap_modules.xml

```diff
@@ -47,8 +53,47 @@ ACL preoperation
     <screen>
 &prompt.sudo;<command>systemctl restart dirsrv@<replaceable>LDAPSERVER1</replaceable>.service</command>
 </screen>
+
+<para>
+ Next, configure the plugin. The following example enables MemberOf to
+ search all entries. Use your instance name rather than the server's
+ hostname:
+</para>
+<screen>&prompt.sudo;<command>dsconf <replaceable>LDAP1 plugin memberOf set --scope dc=example,dc=com</replaceable></command>
+Successfully changed the cn=MemberOf Plugin,cn=plugins,cn=config</screen>
+
+<para>
+ After the MemberOf plugin is enabled and configured, all new groups and
+ users are automatically MemberOf targets. However, any
+ users and groups that exist before it is enabled are not. They must be
+ marked manually:
+</para>
+<screen>&prompt.sudo;<command>dsidm LDAP1 user modify suzanne add:objectclass:nsmemberof</command>
+Successfully modified uid=suzanne,ou=people,dc=ldap1,dc=com</screen>
+
+<para>
+ Now &exampleuserIII;'s information and group membership are listed
+ with a single command:
+</para>
+<screen>&prompt.sudo;<command>dsidm LDAP1 user get &exampleuserIII;</command>
+dn: uid=suzanne,ou=people,dc=ldap1,dc=com
+cn: suzanne
+displayName: Suzanne Geeko
+gidNumber: 102
+homeDirectory: /home/suzanne
+memberOf: cn=SERVER_ADMINS,ou=groups,dc=ldap1,dc=com</screen>
+
+<para>
+ Of course, modifying a larger number of users is tedious. The following
+ example shows how to make all legacy users MemberOf targets with one
+ fixup command:
+</para>
+<screen>&prompt.sudo;<command>dsconf LDAP1 plugin memberof fixup dc=LDAP1,dc=COM</command></screen>
```

Comment:
I think you need "objectClass=*" here on this command (from the top of my head). happy to test and give you the exact command later if you want. Or look at the --help and set that filter.

Reply:
dsconf LDAP1 plugin memberof fixup -f '(objectClass=*)' dc=LDAP1,dc=COM